docker/ubuntu_18.04-build-tools.dockerfile

```diff
@@ -37,8 +37,10 @@
 #
 #   docker run \
 #       --rm \
-#       --mount type=bind,source=$(pwd),target=/home/builder/src \
-#       --user $(id -u):$(id -g) \
+#       --mount type=bind,source=$(pwd),target=/src \
+#       --user root
```

Comment:
The approach taken in this PR requires the entrypoint script to update the user/group id of the `builder` user.
We need to be root to do that.
However, in the end, we want to run the build commands as the `builder` user.

The main issue is that cirrus does not expose a way to specify a user, so it will use the user as specified by the last `USER` statement in the docker file.
Cirrus also does not run the command specified by `ENTRYPOINT`.
That makes it necessary to specify the `builder` user at the end of the docker file.
However, locally, we do want to run the entrypoint script as root to fixup the user/group id in the container to match the user/group id of the host user.
That is why the `--user root` option is necessary at runtime.